### PR TITLE
Store and show AssemblyVersion of connector for each active connection

### PR DESCRIPTION
**Pull request by gingters**

### CODE REVIEW

File: Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs [hunk 1]
```diff
@@ -19,6 +20,13 @@ namespace Thinktecture.Relay.OnPremiseConnector.SignalR
 {
 	internal class RelayServerConnection : Connection, IRelayServerConnection
 	{
+		private const int CONNECTOR_VERSION = 2;
```
Comment (thomashilzendegen) [inline Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs]: The `_` prefix is missing.

File: Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs [hunk 2]
```diff
@@ -28,19 +36,14 @@ internal class RelayServerConnection : Connection, IRelayServerConnection
...
 
-		private static int _nextId;
-		private static readonly Random _random = new Random();
-
-		private const int _MIN_WAIT_TIME_IN_SECONDS = 2;
-		private const int _MAX_WAIT_TIME_IN_SECONDS = 30;
-
 		public RelayServerConnection(string userName, string password, Uri relayServerUri, int requestTimeout, IOnPremiseTargetConnectorFactory onPremiseTargetConnectorFactory, ILogger logger)
-			: base(new Uri(relayServerUri, "/signalr").AbsoluteUri, "version=2")
+			: base(new Uri(relayServerUri, "/signalr").AbsoluteUri, $"version={CONNECTOR_VERSION}&assemblyVersion={Assembly.GetEntryAssembly().GetName().Version}")
```
Comment (thomashilzendegen) [inline Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs]: Let's change the parameter names to `cv` for connector version and `av` for assembly version.

File: Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs (hunk 1 above)
Comment (gingters) [reply Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs]: done

File: Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs (hunk 2 above)
Comment (gingters) [reply Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs]: done

**Pull request merged**